src/analytics/trackers.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { sendEventToHost } from '@wordpress/react-native-bridge';
+
+/**
+ * Internal dependencies
+ */
+import * as EVENTS from './events';
+
+export default {
+	// Track block insertion
+	[ EVENTS.INSERT_BLOCK ]( eventData ) {
+		const { innerBlocks: inner_blocks, name: block_name } = eventData;
+		sendEventToHost( EVENTS.INSERT_BLOCK, { inner_blocks, block_name } );
+	},
```

Comment:
After the event is added to `events.js` add an event handler here to set up the event properties that get sent up to the host apps.